rdr_service/dao/workbench_dao.py

```diff
@@ -1169,6 +1172,30 @@ def _insert_history(session, researchers):
                 affiliations_history.append(affiliation_obj)
             history.workbenchInstitutionalAffiliations = affiliations_history
             session.add(history)
+            hist_researchers.append(history)
+        session.flush()
+        session.commit()
```

Comment:
Is the flush() already an implicit part of the commit(), if you don't need to access anything in between the two calls?

Reply:
I wasn't sure, so I just included it to make sure.